src/app/zap-templates/zcl/data-model/chip/service-area-cluster.xml

```diff
@@ -0,0 +1,243 @@
+
+      <item value="0x20" name="Nightstand"/>
+      <item value="0x21" name="Oven"/>
+      <item value="0x22" name="Pet Bed"/>
+      <item value="0x23" name="Pet Bowl"/>
+      <item value="0x24" name="Pet Crate"/>
+      <item value="0x25" name="Refrigerator"/>
+      <item value="0x26" name="Scratching Post"/>
+      <item value="0x27" name="Shoe Rack"/>
+      <item value="0x28" name="Shower"/>
+      <item value="0x29" name="Side Door"/>
+      <item value="0x2A" name="Sink"/>
+      <item value="0x2B" name="Sofa"/>
+      <item value="0x2C" name="Stove"/>
+      <item value="0x2D" name="Table"/>
+      <item value="0x2E" name="Toilet"/>
+      <item value="0x2F" name="Trash Can"/>
+
+      <item value="0x30" name="Laundry Washer"/>
+      <item value="0x31" name="Window"/>
+      <item value="0x32" name="Wine Cooler"/>
+    </enum>
+
+
+    <enum name="PositionTag" type="enum8">
+      <cluster code="0x0150"/>
+      <item value="0x00" name="Left"/>
+      <item value="0x01" name="Right"/>
+      <item value="0x02" name="Top"/>
+      <item value="0x03" name="Bottom"/>
+      <item value="0x04" name="Middle"/>
+      <item value="0x05" name="Row"/>
+      <item value="0x06" name="Column"/>
+      <item value="0x07" name="Under"/>
+      <item value="0x08" name="Next To"/>
+      <item value="0x09" name="Around"/>
+      <item value="0x0A" name="On"/>
+      <item value="0x0B" name="Above"/>
+      <item value="0x0C" name="Front Of"/>
+      <item value="0x0D" name="Behind"/>
+    </enum>
+
+
+    <enum name="FloorSurfaceTag" type="enum8">
+      <cluster code="0x0150"/>
+      <item value="0x00" name="Carpet"/>
+      <item value="0x01" name="Ceramic"/>
+      <item value="0x02" name="Concrete"/>
+      <item value="0x03" name="Cork"/>
+      <item value="0x04" name="Deep Carpet"/>
+      <item value="0x05" name="Dirt"/>
+      <item value="0x06" name="Engineered Wood"/>
+      <item value="0x07" name="Glass"/>
+      <item value="0x08" name="Grass"/>
+      <item value="0x09" name="Hardwood"/>
+      <item value="0x0A" name="Laminate"/>
+      <item value="0x0B" name="Linoleum"/>
+      <item value="0x0C" name="Mat"/>
+      <item value="0x0D" name="Metal"/>
+      <item value="0x0E" name="Plastic"/>
+      <item value="0x0F" name="Polished Concrete"/>
+
+      <item value="0x10" name="Rubber"/>
+      <item value="0x11" name="Rug"/>
+      <item value="0x12" name="Sand"/>
+      <item value="0x13" name="Stone"/>
+      <item value="0x14" name="Tatami"/>
+      <item value="0x15" name="Terrazzo"/>
+      <item value="0x16" name="Tile"/>
+      <item value="0x17" name="Vinyl"/>
+    </enum>
+
+
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0150" />
+    <field name="ListOrder"       mask="0x01" />
+  </bitmap>
+
+  <!-- Enum definitions for structures-->
+  <enum name="OpStatusEnum" type="enum8">
+    <cluster code="0x0150"/>
+    <item value="0x00" name="Pending"/>
+    <item value="0x01" name="Traversing"/>
```

Comment:
Where is this defined? This dose not seem to be part of the [service area spec PR](https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/8937/files).

Reply:
Yeah, I was referring to the `Traversing` enum only. That's fine.